documentation/src/main/asciidoc/topics/con_protobuf_encoding.adoc

```diff
@@ -0,0 +1,52 @@
+[id='protobuf-encoding_{context}']
+= Protobuf cache encoding
+Protocol Buffers (Protobuf) is a lightweight binary media type for structured data.
+As a cache encoding, Protobuf gives you excellent performance as well as interoperability between client applications in different programming languages and remote server endpoints.
+
+The following example shows a Protobuf message that describes a `Person` object:
+
+[source,json,options="nowrap",subs=attributes+]
+----
+include::protobuf/personalshopper.proto[]
+----
+
+[discrete]
+== Interoperability
+
+Because it is language neutral, Protobuf encoding means {brandname} can handle requests from client applications written in Java, C{plusplus}, C#, Dart, Python, Go, and more.
```

Comment:
I'm guessing you got `Dart` from the google docs? :laughing:. AFAIK it hasn't really taken off outside of Google, so I think we can remove that one.

Reply:
honestly that's an odd one. it must have slipped in there via google docs but the weird thing is that I added the `{plusplus}` attribute and remember looking at this. Dart def doesn't belong here anyway. will remove.